airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/StagingFilenameGenerator.java

```diff
@@ -32,7 +37,7 @@ public StagingFilenameGenerator(final String streamName, final int maxPartsPerFi
    * maxPartsPerFile.
    */
   public String getStagingFilename() {
-    if (currentFileSuffixPartCount < maxPartsPerFile) {
+    if (currentFileSuffixPartCount < CHUNK_LIMIT) {
```

Comment:
could we add tests for this functionality if it doesn't already exist? it may help to restore the constructor parameter (so you can mock it in test)

Reply:
Yep, unit tests are exist and updated. However looks like build wasn't fail during the /test command that's why I didn't update it.